apps/calculation/calculation_store.cpp

```diff
@@ -84,8 +84,13 @@ void CalculationStore::deleteAll() {
 }
 
 void CalculationStore::tidy() {
```

Comment:
m_calculations is a ring buffer, there is no guarantee that m_calculations[numberOfCalculations()] is the last added one.